<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polygonové zdroje na bodové</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLayers">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutput">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOutput">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Procházet</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>77</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Vstup</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOutput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>77</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Výstup</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
